blog-posts/getting-started-gatsby-next-remix.md

```diff
@@ -0,0 +1,306 @@
+    return (
+            <div dangerouslySetInnerHTML={{ __html: post.html }} />
+    );
+}
+```
+
+## Adding Page Metadata
+
+The page meta data, such as the title or description, are especially important for SEO. In the following examples, a title should be added to the pages. For this, each framework offers its own solution.
+
+#### **Gatsby**
+
+To add page metadata in Gatsby, the `gatsby-plugin-react-helmet` and `React Helmet` are recommended. Generally, Gatsby plugins are installed via a package manager and then have to be added to `gatsby-config.js`.
+So in our example blog we first had to run `npm install gatsby-plugin-react-helmet react-helmet` and then add the plugin to `gatsby-config.js`:
+
+```jsx
+{
+  plugins: [`gatsby-plugin-react-helmet`]
+}
+```
+
+After that, you can access the `React Helemet` within the JSX of a component, like for example in the index page.
+
+```tsx
+//...
+import { Helmet } from "react-helmet"
+
+const Index = () => {
+    return (
+        <Layout>
+            <Helmet>
+                <title>Gatsby Blog</title>
+            </Helmet>
+						{/*content goes here*/}
+        </Layout>
+    );
+}
+```
+
+#### **Next.js**
+
+Next.js provides a `head` component that can be built into any page to add metadata.
+
+```tsx
+//...
+import Head from "next/head";
+
+const Home: NextPage = () => {
+    return (
+        <Layout>
+            <Head>
+                <title>Next.js Blog</title>
+								{/*content goes here*/}
+            </Head>
+        </Layout>
+    );
+}
+```
+
+#### **Remix**
+
+To add metadata to Remix projects you have to add the `<Meta />` component in the `<head>` part of `root.tsx`. After that, a `meta` function can be exported to other pages. This must return an object with all relevant metadata. In our example, this is only the page `title`.
+
+```tsx
+//...
+import {MetaFunction} from "remix";
+
+export const meta: MetaFunction = () => {
+    return {
+        title: "Remix Blog",
+    };
+};
+
+export default function Index() {
+  return (
+    <Layout>
+	      {/*content goes here*/}
+    </Layout>
+  );
+}
+```
+
+## Conclusion
+
+Next.js and Gatsby have quite a lot in common as they support both SSR and SSG. The differences are more in detail such as incremental static generation in Next.js or incremental builds in Gatsby. However, it is noticeable that SSR in Gatsby is still a relatively new and not as developed feature as it is in Next.js. The roots of Next.js are in SSR, but relatively soon they also supported SSG. Thus, this framework supports both approaches well.
+
+Remix is the newest of these three frameworks and only supports SSR. However, with the appropriate headers, SSG is approached very closely, which means that it may be possible to completely replace real SSG. The developers of Remix have published an interesting [video](https://www.youtube.com/watch?v=bfLFHp7Sbkg) about this.
+
+![blog-overview.png](images/gastby-next-remix-screenshot-twitter.png)
```

Comment:
I think that's a neat message but focus on the other side of the story:

1. You either manage a running server process plus caching layer and save spend time on optimizing them (SSR)
2. Or you save the time to manage the server and use probably the equal amount of time into your build pipeline where you will struggle to build thousand and thousand of pages at once (SSG).

There is simply no shortcut. I think SSG had its glamour, but I guess we are going full circle and come back to SSR soon. Gatsby introducing SSR capabilities is a first sign of that.

Reply:
As spoken, let's remove the twitter screenshot otherwise the conclusion is great!